scripts/generateAllPlurals.mjs

```diff
@@ -0,0 +1,151 @@
+import {JSDOM} from 'jsdom';
```

Comment:
Can you add a license comment, and also some comment explaining what this script is for so we remember?